apis/aws.go

```diff
@@ -20,15 +20,13 @@ package apis
 import (
 	"k8s.io/apimachinery/pkg/runtime"
 
-	ec2v1alpha1 "github.com/crossplane/provider-tf-aws/apis/ec2/v1alpha1"
-	awsv1alpha1 "github.com/crossplane/provider-tf-aws/apis/v1alpha1"
+	awsv1alpha1 "github.com/crossplane-contrib/provider-tf-aws/apis/v1alpha1"
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
 	AddToSchemes = append(AddToSchemes,
 		awsv1alpha1.SchemeBuilder.AddToScheme,
-		ec2v1alpha1.SchemeBuilder.AddToScheme,
 	)
```

Comment:
I couldn't see the generated schemas added here. Is it something that you plan to handle in a follow-up PR?

Reply:
I did not notice that actually, thank you! It's a bit challenging to do it exactly like we do in other controllers but I have an idea that could make `init` of each CRD package register themselves. Opened https://github.com/crossplane-contrib/terrajet/issues/20 to track.